spec/migrations/20160923000000_remove_replication_excludes_from_settings_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_migration
+
+describe RemoveReplicationExcludesFromSettings do
+  let(:settings_change_stub) { migration_stub(:SettingsChange) }
+  let(:server_one)           { FactoryGirl.create(:miq_server) }
+  let(:server_two)           { FactoryGirl.create(:miq_server) }
+
+  migration_context :up do
+    it "removes the replication excludes key rows" do
+      server_one.settings_changes
+                .create!(:key   => described_class::EXCLUDES_KEY,
+                         :value => %w(schema_migrations))
+      server_two.settings_changes
+                .create!(:key   => described_class::EXCLUDES_KEY,
+                         :value => %w(ar_internal_metadata))
```

Comment:
Minor but you might want one more test of some *other* change to ensure it's not being killed